hax/hax/util.py

```diff
@@ -512,10 +521,11 @@ def get_conf_obj_status(self, obj_t: ObjT, fidk: int) -> int:
             pfid = self.get_service_process_fid(svc_fid)
         else:
             pfid = create_process_fid(fidk)
-        if (self.get_process_status(pfid) in ('M0_CONF_HA_PROCESS_STARTING',
-                                              'M0_CONF_HA_PROCESS_STARTED',
-                                              'M0_CONF_HA_PROCESS_STOPPING',
-                                              'Unknown')):
+        if ((self.get_process_status(pfid)).consul_svc_status in (
+                                                 'M0_CONF_HA_PROCESS_STARTING',
+                                                 'M0_CONF_HA_PROCESS_STARTED',
+                                                 'M0_CONF_HA_PROCESS_STOPPING',
+                                                 'Unknown')):
```

Comment:
[Optional]
```suggestion
        if self.get_process_status(pfid).consul_svc_status in (
                'M0_CONF_HA_PROCESS_STARTING', 'M0_CONF_HA_PROCESS_STARTED',
                'M0_CONF_HA_PROCESS_STOPPING', 'Unknown'):
```
(parentheses are redundant, as a result the code can be more compact)

Reply:
Fixed.